libs/ledger-live-common/src/families/bitcoin/wallet-btc/utils.ts

```diff
@@ -193,3 +193,80 @@ export function writeVarInt(buffer: Buffer, i: number, offset: number): number {
   offset += varuint.encode.bytes;
   return offset;
 }
+
+const strcmp = (a: string, b: string): number => {
+  if (a === b) {
+    return 0;
+  }
+
+  if (a > b) {
+    return 1;
+  }
+
+  return -1;
+};
```

Comment:
Any reason not to use `localeCompare` instead?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare